src/bp/nlu-core/entities/microsoft-extractor/typings.ts

```diff
@@ -0,0 +1,219 @@
+  Recognizers.recognizeNumber,
+  Recognizers.recognizePercentage,
+  Recognizers.recognizeAge,
+  Recognizers.recognizeCurrency,
+  Recognizers.recognizeDimension,
+  Recognizers.recognizeTemperature,
+  Recognizers.recognizeDateTime,
+  Recognizers.recognizeBoolean
+]
+
+export const DucklingUnitMapping = {
+  volume: [
+    'Barile',
+    'Millilitre',
+    'Baril',
+    'Tablespoon',
+    'Gallon',
+    'Barril',
+    'Volume unit',
+    'Litro',
+    'Cubic meter',
+    'Pé cúbico',
+    'Litre',
+    'Liter',
+    'Piede cubo',
+    'Cubic foot',
+    'Galón',
+    'Centímetro cúbico',
+    'Millilitro',
+    'Pieds cube',
+    'Teaspoon',
+    'Onces',
+    'Pie cúbico',
+    'Milliliter',
+    'Gallone',
+    'Galão',
+    'Mililitro'
+  ],
+  distance: [
+    'Foot',
+    'Pé',
+    'Picometer',
+    'Millimètres',
+    'Kilometer',
+    'Millimetro',
+    'Pie',
+    'Meter',
+    'Micrometer',
+    'Mile',
+    'Pollice',
+    'Light year',
+    'Mètres',
+    'Centimètres',
+    'Millimeter',
+    'Milímetro',
+    'Hectomètre',
+    'Décimètres',
+    'Miglio',
+    'Inch',
+    'Metro',
+    'Quilômetro',
+    'Milha',
+    'Nanometer',
+    'Milla',
+    'Kilómetro',
+    'Pulgada',
+    'Polegada',
+    'Pied',
+    'Kilomètres',
+    'Pouce',
+    'Piede',
+    'Chilometro'
+  ],
+  weight: [
+    'Ounce',
+    'Oncia',
+    'Libra',
+    'Metric ton',
+    'Gallon',
+    'Pound',
+    'Kilogram',
+    'Tonelada',
+    'Tonne métrique',
+    'Ton',
+    'Barrel',
+    'Tonelada métrica',
+    'Onza',
+    'Libbra',
+    'Gram',
+    'Tonnellata',
+    'Onça',
+    'Tonne',
+    'Livre'
+  ],
+  area: [
+    'Mètre carré',
+    'Metro cuadrado',
+    'Square decameter',
+    'Square hectometer',
+    'Square millimeter',
+    'Chilometro quadrato',
+    'Acro',
+    'Quilômetro quadrado',
+    'Square kilometer',
+    'Square meter',
+    'Kilómetro cuadrado',
+    'Square decimeter',
+    'Acre',
+    'Metro quadrato',
+    'Square centimeter',
+    'Metro quadrado',
+    'Hectare',
+    'Kilomètre carré'
+  ],
+  bits: ['Kilobyte', 'Terabyte', 'bit', 'Gigabyte', 'Megabit', 'Megabyte', 'Bit'],
+  speed: [
+    'Miles par heure',
+    'Kilometer per hour',
+    'Milla por hora',
+    'Meter per second',
+    "Miglia all'ora",
+    'Mile per hour',
+    'Milha por hora'
+  ]
+}
+
+export const DucklingTypeMappings = {
+  currency: 'amountOfMoney',
+  email: 'email',
+  number: 'number',
+  ordinal: 'ordinal',
+  phonenumber: 'phoneNumber',
+  temperature: 'temperature',
+  url: 'url'
+}
+
+export const DucklingDateMappings = {
+  // https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-reference-prebuilt-datetimev2?tabs=1-3%2C2-1%2C3-1%2C4-1%2C5-1%2C6-1
+  'datetimeV2.datetimerange': 'duration',
+  'datetimeV2.daterange': 'duration',
+  'datetimeV2.timerange': 'duration',
+  'datetimeV2.duration': 'duration',
+  'datetimeV2.date': 'time',
+  'datetimeV2.datetime': 'time',
+  'datetimeV2.time': 'time'
+}
```

Comment:
The name of the file doesn't correspond to its content, in a "typings" file I would expect interfaces and types. I think `enums.ts` would be more appropriate